plugins/cache_promote/cache_promote.cc

```diff
@@ -134,6 +186,29 @@ class ChancePolicy : public PromotionPolicy
   {
     return "chance";
   }
+
+  bool
+  stats_add(const char *remap_id) override
+  {
+    std::string_view remap_identifier = remap_id;
+
+    std::string_view stats[3] = {"cache_hits", "promoted", "total_requests"};
+
```

Comment:
Ditch the empty line, and perhaps it should be const std::string_view ... ?

Reply:
done and done